frontend/pages/beta/model/[modelId]/schema.tsx

```diff
@@ -34,9 +37,17 @@ export default function NewSchemaSelection() {
 
   async function createModelUsingSchema(newSchema: SchemaInterface) {
     if (currentUser && model) {
-      await postFromSchema(model.id, newSchema.id)
-      await mutateModel()
-      router.push(`/beta/model/${modelId}`)
+      setLoading(true)
+      setErrorMessage('')
+
+      const response = await postFromSchema(model.id, newSchema.id)
+
+      if (response.status && response.status < 400) {
```

Comment:
Should this not be if(!response.ok) ?

Reply:
It's an axios request so the response doesn't have an `ok` property. The `response.statusText` will be `'OK'` if the `response.status` is `200`, but we parse the response through `handleAxiosError` which ignores the `statusText`. If we want to change this I'd argue we can easily do it later on.